### ISSUE TITLE
Integrate Testing into Monorepo

### ISSUE THREAD

**Issue by chrisfalaska** [AlaskaAirlines/auro-formkit#85]
### Please verify the version of auro-form you have installed

1.5.0

### Please describe the bug

In the new monorepo structure, running `npm run test` fails.

For independent component packages that do not have dependencies on other component packages in `auro-formkit`, the tests pass.

However, for components with `auro-formkit` dependencies, such as `combobox`, the tests will fail:

```
Error while generating code coverage for http://localhost:8000/__wds-outside-root__/1/input/dist/index.js.
@auro-formkit/auro-datepicker:test: Error: ENOENT: no such file or directory, open '/Users/AlaskaUser/Dev/auro-form/components/datepicker/__wds-outside-root__/1/input/dist/index.js'
```

The issue seems to be with the URL Web Test Runner generates, specifically:

```
__wds-outside-root__
```

This may be related to these issues:

- https://github.com/modernweb-dev/web/issues/2681
 
- https://github.com/modernweb-dev/web/issues/1536

### Expected behavior

`wtr` should not fail when running tests that depend on other component packages within `auro-formkit`.

### Additional context

#19

**Comment by chrisfalaska** [comment]
Currently blocked by #19 until merged.

**Comment by chrisfalaska** [comment]
Be sure to return `test` to the `.husky/pre-commit` file:

```
./node_modules/.bin/npm-run-all preCommit test lint postinstall
```

**Comment by jordanjones243** [comment]
Interesting observation while trying to repro this... auro-select, which is dependent on both menu, passes it's test. Funnily enough, that same error message pops up during the process. Hard to say whether the error message listed in the issue is the actual problem with combobox and datepicker.

**Comment by jordanjones243** [comment]
After further investigation, the error listed in this ticket is not what is causing the tests to fail. Upon further investigation, one of there are bugs causing datepicker and combobox's tests to fail and will be addressed in a separate ticket. The work in this ticket will be to track down what is causing the `wds-root` errors at a later time.
